### ISSUE TITLE
PreparedStatement recognizes UPDATE statement as INSERT when calling .getParameterMetaData()

### ISSUE THREAD

**Issue by jpoernomoatsafe** [olavloite/spanner-jdbc#16]
Code snippet:

```
String sqlStatement = "UPDATE `MY_TABLE` SET `MY_NUM` = ?, `MY_STRING` = ? WHERE `MY_ID` = ?"
PreparedStatement preparedStatement = cloudSpannerConnection.prepareStatement(sqlStatement)
preparedStatement.getParameterMetaData()
```

the last line will throw an Exception with this message:
```
java.sql.SQLException: Error while parsing sql statement INSERT INTO `MY_TABLE`
(, `MY_NUM`, `MY_STRING`)
SELECT , ?, ?
FROM `MY_TABLE`
WHERE `MY_ID` = ?
ON DUPLICATE KEY UPDATE: null
	at COM.safe.fme.jdbc.JdbcBackendHelper.bindValues(JdbcBackendHelper.java:1233)
	at COM.safe.fme.jdbc.BatchedStatements.addToBatch(BatchedStatements.java:69)
	at COM.safe.fme.jdbc.JdbcWriter.batchStatementAndExecute(JdbcWriter.java:763)
	at COM.safe.fme.jdbc.JdbcWriter.updateFeature(JdbcWriter.java:732)
	at COM.safe.fme.jdbc.JdbcWriter.writeFeature(JdbcWriter.java:587)
	at COM.safe.fme.jdbc.JdbcWriter.write(JdbcWriter.java:280)
```

Expected:
update statement didn't magically turn into insert statement. 

when debugging, I was able to confirm that `preparedStatement.sql` had the right SQL statement. 

v0.15 didn't have this issue. v0.16 was the first version with this issue

**Comment by olavloite** [comment]
This bug was introduced with the introduction of support for bulk update statements.

**Comment by olavloite** [comment]
The bug was introduced with the introduction of support for bulk update statements. These are internally translated into INSERT INTO ... ON DUPLICATE KEY UPDATE statements, and this translation also caused the meta data to be lost. It has been fixed for version 0.20.
